sample/shared/shared/src/commonMain/kotlin/com/arkivanov/sample/shared/multipane/list/PreviewArticleListComponent.kt

```
@@ -0,0 +1,24 @@
+    override val models: Value<Model> =
+        MutableValue(
+            Model(
+                articles = List(10) { index ->
+                    Article(id = index.toLong(), authorId = index.toLong(), title = "Article $index")
+                },
+                isToolbarVisible = isToolbarVisible,
+                selectedArticleId = null,
+            )
+        )
+
+    override fun onArticleClicked(article: Article) {}
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)